python/ray/serve/config.py

```diff
@@ -214,6 +214,7 @@ class DeploymentMode(str, Enum):
     NoServer = "NoServer"
     HeadOnly = "HeadOnly"
     EveryNode = "EveryNode"
+    FixedNumber = "FixedNumber"
```

Comment:
Hmm I'm pretty sure that DeploymentMode shows up in the API docs -- is there any way to hide this option?

Reply:
![image](https://user-images.githubusercontent.com/21118851/134081399-f6dd3237-88d0-469a-9602-3088c3239318.png)
they are not in fact. Because we hand wrote the doc in `serve.start` doc string and I purposely didn't document it there.